app/scripts/components/common/page-footer/styles.scss

```diff
@@ -1,7 +1,72 @@
 @use '$styles/veda-ui-theme-vars.scss' as themeVars;
+
+#logo-container {
```

Comment:
I don't think we want all the `#logo-container` styles here. I am guessing you only meant to add the styles for `#nasa-logo-pos`?

Reply:
That is correct this is a hold over from the header work I copied in to get the logo to preview.